### ISSUE TITLE
URLs don't work using Laragon as local server

### ISSUE THREAD

**Issue by ivanblue** [ridwanskaterock/cicool#109]
Hello, I'm using Laragon as a local server (it's like XAMPP) and though I was able to install CICool, after the installation URLs show 404 Page not found and I can't login, but I'm able see the home page. It's really weird. Do you have any ideas about why this happens and how can I make it work using Laragon?

Thanks in advance. Best regards.

**Comment by ridwanskaterock** [comment]
can you give me screenshoot for that ? 

Whether when you open the installation url by adding index.php will appear?

thanks

**Comment by ivanblue** [comment]
Hello, installation works. The application it's installed and the db it's populated, but after that it doesn't work anymore. Home and login shows, but I cannot login with my email and password.

This is Home, which shows without any issues
![sin-titulo-1](https://user-images.githubusercontent.com/5141755/29776387-7901bc32-8bd6-11e7-93ca-5f74da9e7bd0.jpg)
about page shows 404
![sin-titulo-2](https://user-images.githubusercontent.com/5141755/29776427-96fb12f6-8bd6-11e7-9e75-f5d3f55051dc.jpg)

Even when I put index.php in front
![sin-titulo-3](https://user-images.githubusercontent.com/5141755/29776473-b3776b82-8bd6-11e7-9b72-9af784932ef7.jpg)

Laragon creates a virtualhost in apache and modifies windows host file so I can use "cicool.dev" as my development url. This is the added code in windows host file (in case it helps):

127.0.0.1      cicool.dev

Thanks for your attention and prompt response.

**Comment by ridwanskaterock** [comment]
hiii 

for page about it's for example only you can make page with page builder and

for login what your password is correct ? can you give me the error response ? 

thanks

**Comment by ivanblue** [comment]
Before I installed laragon, I installed CICool on a Bitnami WAMP and it works perfectly fine. The about page and the login worked. But now with Laragon something messed up with the CICool routes (or so I think).

When I tried to login there's no error message. It just sends me to the home page without loggin me in. 

Any idea about what could be wrong?

I'm using Laragon with other sites using CodeIgniter without any problems.

Thanks for your time. Best regards.
